src/utilities/Makefile

```diff
@@ -30,6 +30,7 @@ HEADERS =\
  mpistubs.h\
  threading.h\
  timing.h\
+ _hypre_onedpl.hpp\
```

Comment:
I'm trying to remember if this is how we handle some of the other GPU ports.  Maybe @liruipeng can comment.

Reply:
I was wondering if this file could go to `_hypre_utilities.hpp`. That's how we handled other cpp headers.

Reply:
These headers are a bit of a strange special case. The oneDPL headers must be included before any C++ standard library headers in order to function correctly. I was having trouble finding an appropriate place in our current header setup that would yield the correct behavior (in particular, putting these in _hypre_utilities.hpp does not work), so I put them in their own header file that can be included at the very top of any source file where they are needed (e.g. seq_mv/vector.c). I'd be happy to hear thoughts on alternative ways of doing this.

Reply:
I don't have any bright ideas off the top of my head.  We can revisit this later.

Reply:
I don't know much about about oneDPL. So even if you put it at the top of `_hypre_utilities.hpp`, it still did not work? It may be fine to me, just carrying another header file in `hypre/include`. Maybe @rfalgout can comment more. Thanks @waynemitchell .

Reply:
It's unclear to me whether this will always be true of the oneDPL headers or if they will be more flexible in the future. But for now, this was the cleanest quick solution I came up with. I think we can leave this as is for now and revisit later if oneDPL changes or I figure out a nicer way to include them.

Reply:
Sounds good to me. Thanks!